shared/versions.yml

```diff
@@ -29,12 +29,12 @@ cloud_sdk:
     tag: 242.0.0-alpine
 couchdb_helper:
   upstream:
-    repository: kocolosk/couchdb-statefulset-assembler
-    tag: 1.2.0
+    repository: gpii/couchdb-statefulset-assembler
+    tag: 1.3.0-gpii.0
   generated:
-    repository: gcr.io/gpii-common-prd/kocolosk__couchdb-statefulset-assembler
+    repository: gpii/couchdb-statefulset-assembler
```

Comment:
I would suggest leaving out changes in the `generated:` section in future, this typically causes merge conflicts if you're not fast enough merging the PR ;) (and is not necessary, if anything it can cause broken build if this would be picked up by CI before version-updater kicks in).

Reply:
This allows to test the changes with updated image. Pipeline will be broken anyway if it starts before version-updater commit... But yeah, changes to `generated` section does not need to be committed.